spec/helpers/text_formatting_helper_spec.rb

```diff
@@ -24,7 +24,7 @@
       let(:html) do
         <<~HTML
           <div>test</div>
-          <p>
+          <p id="paragraph">
```

Comment:
I suppose `id` could be added to line 66 just to ensure that it gets stripped out ?